src/Propel/Generator/Reverse/OracleSchemaParser.php

```diff
@@ -228,6 +224,24 @@ protected function addIndexes(Table $table): void
         }
     }
 
+    /**
+     * @param \Propel\Generator\Model\Table $table
+     *
+     * @return array<scalar, non-empty-list<null|scalar>>
+     */
+    protected function getColumnNamesIndexedByIndexName(Table $table): array
+    {
+        /** @var \PDOStatement $stmt */
+        $stmt = $this->dbh->query("SELECT INDEX_NAME, COLUMN_NAME FROM USER_IND_COLUMNS WHERE TABLE_NAME = '" . $table->getName() . "' ORDER BY COLUMN_NAME");
+
+        $columnNamesIndexedByIndexName = [];
+        while ($row = $stmt->fetch(PDO::FETCH_ASSOC)) {
```

Comment:
why don't we need the `if ($stmt === false)` check here?

Reply:
I guess the while truish implicist cast takes care of this?